### ISSUE TITLE
Allow Project Experts and/or Project Researchers access

### ISSUE THREAD

**Issue by shaunanoordin** [zooniverse/messenger#4]
## Request: Overview
* At the moment, only Project Owners and Project Collaborators are allowed write access to the Messenger service.
* In practice (notably with the AnnoTate project), we realised that our expert users may need to share their work with other team members, who may not/should not have all the elevated privileges of a Collaborator. _(i.e. someone who's just reviewing documents should not be able to delete the project)_
* Hence, it may make sense to give read access in `messenger` to less-privileged Panoptes roles, e.g. Project Expert and/or Project Researcher.
  * The Project Expert role may be the most appropriate in this context: _"Sarah: Expert probably makes the most sense, since that's the role that lets you submit gold standard classifications"_
* For the full discussion, please check Slack, #dev channel, from 13 Jul 2017 17:06 BST to 17:37BST

**Comment by shaunanoordin** [comment]
Regarding priority:
* From the user & front-end perspective, giving the Expert role write access is not a high priority item, as the AnnoTate Transcription Viewer project may proceed with the current "everyone gets to be a Collaborator!" method.
  * We can also later downgrade roles from Collaborator to Expert, when the changes are implemented.
* However, I'm aware that from a security-best-practice viewpoint, there is a strong argument for "don't give people more access than necessary from the get go" and this issue of roles might be more important than I realise.

In short, I'm not going to ask that this upgrade be rushed; however, if it _is_ a _huge_ security concern, and that we shouldn't be handing out Collaborator roles like candy, please let me know and I'll tell Tate to wait while the upgrade gets put in place.

**Comment by zwolf** [comment]
Users with the `expert` role on the Panoptes project can now create and update Messenger transcriptions. Closed by https://github.com/zooniverse/messenger/pull/5.
